### ISSUE TITLE
add action step to publish new version to https://pypi.org/ 

### ISSUE THREAD

**Issue by tb102122** [mingrammer/diagrams#1051]
Hey @mingrammer,
let me know if we should add an action to add a publishing step,

**Comment by mingrammer** [comment]
Yes, we have already planned to add a step for publishing the package and website automatically when a new tag version is released.

**Comment by tb102122** [comment]
ok let me know if I can help we we have the code more less ready in the other repo.

**Comment by mingrammer** [comment]
Sure. Thank you :)
